### PR TITLE
Add new OGM and AGOL pallets

### PR DESCRIPTION
**Pull request by stdavis**
I've tested both of these pallets.

`sgid_secrets.py` is already on the server along with all required database connections in the garage.

### CODE REVIEW

File: sgid/OGMPallet.py [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+            arcpy.management.DeleteRows(destination)
+            arcpy.management.Append(source, destination)
+
+        if (self.configuration == 'Production'):
+            self.update_ftp_package()
+
+    def update_ftp_package(self):
+        self.log.info('updating ftp package')
+        name = 'DOGMOilAndGasResources'
+        packageFolderPath = r'\\' + secrets.HNAS + r'\ftp\UtahSGID_Vector\UTM12_NAD83\ENERGY\PackagedData\_Statewide\\' + name
```
Comment (steveoh) [inline sgid/OGMPallet.py]: should this have the double slash at the end or is that oging to expand too many times?

File: sgid/OGMPallet.py [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+        unpackagedFolderPath = r'\\' + secrets.HNAS + r'\ftp\UtahSGID_Vector\UTM12_NAD83\ENERGY\UnpackagedData\\'
```
Comment (steveoh) [inline sgid/OGMPallet.py]: same with this one?

File: sgid/OGMPallet.py [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+    import logging
+
+    pallet = OGMPallet()
+    logging.basicConfig(
+        format='%(levelname)s %(asctime)s %(lineno)s %(message)s',
+        datefmt='%H:%M:%S',
+        level=logging.INFO
+    )
+    pallet.log = logging
+    pallet.build('Dev')
```
Comment (steveoh) [inline sgid/OGMPallet.py]: should this have dev in it or should it use the forklift configuration?

Comment (stdavis) [inline sgid/OGMPallet.py]: This is for running the module manually outside of forklift for testing purposes. I believe that 'Dev' is the most common option in this case.

File: sgid/OGMPallet.py (hunk 1 above)
Comment (stdavis) [inline sgid/OGMPallet.py]: It needs the double slash or there's a syntax error. With only one slash it thinks that it's escaping the end quote.

**Pull request merged**